run_tests.py

```diff
@@ -261,10 +261,15 @@ def test_from_failstr(failstr):
 
 # Redirect stderr into a file
 tee = subprocess.Popen(
```

Comment:
We must remove this process because now `FileHandler` write logs to file or we have duplicate log entries

Reply:
Will do it.